plugins/plugin-optimize/README.md

```diff
@@ -29,10 +29,12 @@ Then add this plugin to your Snowpack config:
-| `target`         | `string,string[]` | The language target(s) to transpile to. This can be a single string (ex: "es2018") or an array of strings (ex: ["chrome58","firefox57"]). If undefined, no transpilation will be done. See [esbuild documentation](https://github.com/evanw/esbuild) for more. |
+| Name               |       Type        | Description                                                                                                                                                                                                                                                    |
+| :----------------- | :---------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `minifyJS`         |     `boolean`     | Enable JS minification (default: `true`)                                                                                                                                                                                                                       |
+| `minifyCSS`        |     `boolean`     | Enable CSS minification (default: `true`)                                                                                                                                                                                                                      |
+| `minifyHTML`       |     `boolean`     | Enable HTML minification (default: `true`)                                                                                                                                                                                                                     |
+| `preloadModules`   |     `boolean`     | Experimental: Add deep, optimized [`<link rel="modulepreload">`](https://developers.google.com/web/updates/2017/12/modulepreload) tags into your HTML. (default: `false`)                                                                                      |
+| `preloadCSS`       |     `boolean`     | Experimental: Eliminate `.css.proxy.js` files and combine imported CSS into one file for better network performance (default: `false`)                                                                                                                         |
+| `preloadedCSSName` |     `string`      | If preloading CSS, change the name of the generated CSS file. Only used in conjunction with `preloadCSS: true` (default: `/imported-styles.css`)                                                                                                               |
```

Comment:
nit: `Name` makes me think some sort of CSS class or variable name. We use `FileName` and `FileLoc` in other plugins, that may be a better fit here

Reply:
trying out `preloadCSSFileName`